extension/src/experiments/index.ts

```diff
@@ -199,6 +201,15 @@ export class Experiments extends BaseRepository<TableData> {
     return this.notifyChanged()
   }
 
+  public setPushing(ids: string[]) {
+    this.experiments.setPushing(ids)
+    return this.notifyChanged()
+  }
+
+  public unsetPushing(ids: string[]) {
+    this.experiments.unsetPushing(ids)
```

Comment:
[F] We don't notify here because we want to fetch the data again.

Reply:
I've moved `update` into here to make sure the function is never accidentally called without it.